UI/obs-app.cpp

```diff
@@ -1294,6 +1296,8 @@ OBSApp::~OBSApp()
 	os_inhibit_sleep_set_active(sleepInhibitor, false);
 	os_inhibit_sleep_destroy(sleepInhibitor);
 
+	delete shortcutFilter;
```

Comment:
Probably a good idea to use QScopedPointer, so we don't have to manually delete the shortcut filter.

Reply:
Not really sure how to do this, do you have an example? I tried `QScopedPointer<QObject> shortcutFilter;` which seemed correct from the documentation but it does not compile. It works if I explicitly call `shortcutFilter.data());` but according to the docs that shouldn't be necessary?
```
error C2664: 'void QObject::installEventFilter(QObject *)': cannot convert argument 1 from 'QScopedPointer<QObject,QScopedPointerDeleter<T>>' to 'QObject *'
```

Reply:
Afaik that's necessary - you either have to use `data()`, `take()`, or you pass a reference to the pointer (which would be `installEventFilter(&App()->shortcutFilter);`) when using a `QScopedPointer`.